copier/cli.py

```diff
@@ -358,6 +367,59 @@ def main(self, destination_path: cli.ExistingDirectory = ".") -> int:
         return 0
 
 
+@CopierApp.subcommand("check")
+class CopierCheckSubApp(cli.Application):
+    """The `copier check` subcommand.
+
+    Use this subcommand to check if an existing subproject is using the
+    latest version of a template.
+
+    Attributes:
+        exit-code: Set [exit-code][] option.
+    """
+
+    DESCRIPTION = "Check if a copy is using the latest version of its original template"
+    DESCRIPTION_MORE = dedent(
+        """\
+        The copy must have a valid answers file which contains info
+        from the last Copier execution, including the source template
+        (it must be a key called `_src_path`).
+
+        If that file contains also `_commit` and `destination_path` is a git
+        repository, this command will do its best to determine whether a newer
+        version is available, applying PEP 440 to the template's history.
+        """
+    )
+
+    exit_code: cli.SwitchAttr = cli.SwitchAttr(
+        ["-e", "--exit-code"],
+        int,
+        default=0,
+        help=(
+            "Exit code for the command if a newer version is available, "
+            "for use in scripts."
+        ),
+    )
+
```

Comment:
I don't think this should be customizable. Instead, it should be documented and Copier should be the one keeping control over which exit codes it uses.